demo-extended/src/main/java/com/novoda/simplechromecustomtabs/demo/linkify/MatchedUrl.java

```diff
@@ -0,0 +1,15 @@
+package com.novoda.simplechromecustomtabs.demo.linkify;
+
+class MatchedUrl {
+
+    final String url;
+    final int start;
+    final int end;
+
+    MatchedUrl(String url, int start, int end) {
+        this.url = url;
+        this.start = start;
+        this.end = end;
+    }
+
```

Comment:
Done in https://github.com/novoda/simple-chrome-custom-tabs/pull/21/commits/eae6245a2b0f8c44a03b807594cc948e3d5ff55e